src/analysis/souffle/policy_verifier_interface.dl

```diff
@@ -20,6 +20,7 @@
 #include "src/analysis/souffle/sql_output.dl"
 #include "src/analysis/souffle/taint.dl"
 #include "src/analysis/souffle/tag_transforms.dl"
+#include "src/analysis/souffle/epsilon_analysis.dl"
```

Comment:
Is this include necessary here at this time?

Reply:
We are checking for violation of policy, so yes.